### ISSUE TITLE
About training details

### ISSUE THREAD

**Issue by MorenMoren** [buaa-colalab/UAV-Flow#5]
Thanks for your open source datasets and your great work.

I have tried to finetune the openvla-oft using uav-flow datasets, however the loss is still high:

<img width="3604" height="1350" alt="Image" src="https://github.com/user-attachments/assets/f96900db-90a4-4a58-b2c4-8e85ddbf46f3" />

I would like to know whether the "action" in the training data uses the pose relative to the starting point coordinate system or the world coordinate system during training, and what the “state” in training data is like.

**Comment by prince687028** [comment]
Hello! The action represents the displacements over the next few time steps under the current coordinate system, and these displacements have been normalized.
The state corresponds to the coordinates expressed relative to the starting point coordinate system.

**Comment by fulanya55** [comment]
@MorenMoren May I ask if you finally ran it using the OFT architecture? I am also trying to reproduce it with OFT, but I feel that OFT is not very suitable for this kind of task. Would using Pi0 be a bit better?
